Artifacts/linux-syscoin/Artifactfile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Syscoin (Business on the blockchain)",
```

Comment:
We like to have simplified titles. Will just "Syscoin" do? More clarification can go into the description.